indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -1297,7 +1298,7 @@ public Response getPhaseName(@Context final HttpServletRequest req)
         return Response.ok(runner.getName()).build();
       }
     } else {
-      return Response.status(Status.BAD_REQUEST).entity("task is running in the sequential mode").build();
+      return ResponseStatusExceptionMapper.toResponse(Response.Status.BAD_REQUEST, "task is running in the sequential mode");
```

Comment:
I cleaned up all the BAD_REQUEST response code to use the same response format.

Reply:
I think it's safe to do this because we don't know in the future if the string content will be changed. If it's changed, there's a risk to experience the same problem we're facing now.

Actually, the original API reported by the author is also a user facing API, changing the response seems incompatibility. But I think it's OK because these changes happen in error cases which are rare.

Reply:
Perhaps I should have been clearer. All I'm worrying about is the error message. It should not change unless there is a good reason especially for user-facing APIs.

Reply:
Let me make my stance clearer here. I think `ResponseStatusExceptionMapper` is a good idea to prevent future mistakes of missing content type. It's worth modifying user-facing API responses. To make sure that we always use the nice helper method instead of making a `Response` on our own, we should probably register `Response.status()` or `Response.serverError()` as forbidden APIs. If we are going down that way, we should change all user-facing APIs in the same release so that we won't introduce similar incompatible changes over multiple releases. Perhaps you or someone else is planning to do this in multiple PRs, but because things don't always go as planned, I think it should be better to do in one PR. But this is going to be a huge refactoring, so I would suggest to do this separately from the original security issue.

Reply:
I intendend to forbidden the call of `Response.status` or `Response.serverError()`. 

As you said, there are too many references we need to modify, so I didn't do all the things in this patch, but just modified all BAD_REQUEST, which was the same error as the original report, and 3 or 4 places of NOT_FOUND error.